.changeset/silent-coats-check.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": major
+"@khanacademy/perseus-score": major
```

Comment:
What makes this a major change? The extent of the change?

Reply:
We are changing the name of types that are exported in a non-backwards compatible way. According to semver, this is a major change:

> 1. MAJOR version when you make incompatible API changes
> 2. MINOR version when you add functionality in a backward compatible manner
> 3. PATCH version when you make backward compatible bug fixes